internal/mode/static/state/graph/gatewayclass.go

```diff
@@ -75,11 +76,22 @@ func buildGatewayClass(gc *v1beta1.GatewayClass) *GatewayClass {
 	}
 }
 
-func validateGatewayClass(gc *v1beta1.GatewayClass) error {
+func validateGatewayClass(gc *v1beta1.GatewayClass, npCfg *ngfAPI.NginxProxy) error {
 	if gc.Spec.ParametersRef != nil {
-		path := field.NewPath("spec").Child("parametersRef")
-		return field.Forbidden(path, "parametersRef is not supported")
+		if _, ok := supportedParamKinds[string(gc.Spec.ParametersRef.Kind)]; !ok {
```

Comment:
Should we also check that the namespace is not nil in the parameter ref since our proxy CRD is namespaced?

Reply:
Yep good call